### ISSUE TITLE
Automatically generated Order Number

### ISSUE THREAD

**Issue by HeinzWuert** [kevinpapst/kimai2#3389]
**Is your feature request related to a problem? Please describe.**
Would like to see the same feature of generated invoice number for order number.

Nothing more to say.
Thanks a lot in advance :-)

**Comment by kevinpapst** [comment]
Which replacers would you need for the format?

**Comment by HeinzWuert** [comment]
we use a combination of

- date: yymmdd
- a daily counter, one sign
- the first two letters of customer
- the first letter of a metafield (see #3382)
- separators  "-"

Looks like:  220511-1-Ha-A

**Comment by HeinzWuert** [comment]
Isn't it the same like realized #3373 ?

**Comment by kevinpapst** [comment]
No. While it seems to be common sense to use a "customer number", I do not know any other customers that uses the order number. 
And looking at your format of that number, I strongly doubt that Kimai will be able to generate that.
